pkg/config/run.go

```diff
@@ -23,7 +23,7 @@ type Run struct {
 	AllowParallelRunners bool `mapstructure:"allow-parallel-runners"`
 	AllowSerialRunners   bool `mapstructure:"allow-serial-runners"`
 
-	ShowStats bool `mapstructure:"show-stats"`
+	ShowStats bool `mapstructure:"show-stats"` // Deprecated use Output.ShowStats instead.
```

Comment:
```suggestion
	// Deprecated: use Output.ShowStats instead.
	ShowStats bool `mapstructure:"show-stats"`
```

Did you intentionally write the [deprecated comment](https://go.dev/wiki/Deprecated) incorrectly to prevent `staticcheck` from raising the issue `SA1019`?

Reply:
It was just a mistake.